locales/fr.json

```diff
@@ -305,12 +321,17 @@
     },
     "aria-required-children": {
       "pass": "Les descendants ARIA requis sont présents",
-      "fail": "{{=it.data && it.data.length > 1 ? 'Descendants ARIA requis absents' : 'Descendant ARIA requis absent'}} : {{~it.data:value}} {{=value}}{{~}}"
+      "fail": "{{=it.data && it.data.length > 1 ? 'Descendants ARIA requis absents' : 'Descendant ARIA requis absent'}} : {{~it.data:value}} {{=value}}{{~}}",
+      "incomplete": "{{=it.data && it.data.length > 1 ? 'Les descendants' : 'Le descendant'}} requièrent le rôle ARIA : {{~it.data:value}} {{=value}}{{~}}"
```

Comment:
_requièrent_ is correct with plural but not otherwise.
`'Les descendants' : 'Le descendant'}} requièrent` should become something like:  
`'Les descendants requièrent' : 'Le descendant requiert'}} `